packages/e2e-tests/specs/experiments/document-settings.test.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	visitAdminPage,
+	trashAllPosts,
+	activateTheme,
+} from '@wordpress/e2e-test-utils';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { navigationPanel } from '../../experimental-features';
+
+async function getDocumentSettingsTitle() {
+	const titleElement = await page.waitForSelector(
+		'.edit-site-document-actions__title'
+	);
+
+	return titleElement.evaluate( ( el ) => el.innerText );
+}
+
+async function getDocumentSettingsSecondaryTitle() {
+	const secondaryTitleElement = await page.waitForSelector(
+		'.edit-site-document-actions__secondary-item'
+	);
+
+	return secondaryTitleElement.evaluate( ( el ) => el.innerText );
+}
+
+describe( 'Document Settings', () => {
+	beforeAll( async () => {
+		await activateTheme( 'tt1-blocks' );
+		await trashAllPosts( 'wp_template' );
+		await trashAllPosts( 'wp_template_part' );
+	} );
+	afterAll( async () => {
+		await trashAllPosts( 'wp_template' );
+		await trashAllPosts( 'wp_template_part' );
```

Comment:
I noticed we are dropping these two statements when we aren't making any changes to entities.  Maybe we are safe to drop them here too?

https://github.com/WordPress/gutenberg/blob/37e15797f2259ee0661b56b5a8f877bde9f29099/packages/e2e-tests/specs/experiments/multi-entity-editing.test.js#L126-L128

Reply:
That makes sense 👍